_posts/2019-02-24-The-Intelligent-Investor.md

```diff
@@ -0,0 +1,152 @@
+markets, etc.
+
+**I will not help you with individual financial stuff. Please do not ask.** Consult with a fiduciary advisor with any specific question you have and make sure your 401k company is not screwing you over. *Hint*: They are probably screwing you over, watch [The Retirement Gamble](https://www.pbs.org/video/frontline-retirement-gamble/).
+
+
+### The Gist
+
+_The Intelligent Investor_ and _Security Analysis_ are the only investment books I ever needed to read, all the others were further elaboration on Benjamin Graham's points. I decided to do this book review because his principles to preserve wealth, reducing the amount of false positives, and truly invest for the long run <sup>1</sup> are timeless techniques. For the purposes of brevity, I will focus on what Benjamin Graham calls a _Defensive Investor_ instead of the _Enterprising Investor_.
+
+Intelligent investing is not about being the smartest person in the room, it is about following sound principles to their ends. An intelligent investor is someone who develops temperance, patience, and control. She is learned, meticulous in their reasoning, and defensive in their thinking. She becomes familiar with the major ideas in many fields related to investing. These fields include, but are not limited to: psychology, history, basic statistics, meditation, and human evolution. An intelligent investor has more in common with an endurance athlete than a superstar sports player, in this game consistency can beat talent in many cases.
+
+One of the worst investors of all time is Sir Isaac Newton. I don't think many people would consider him stupid, but he was a stupid investor.<sup>2</sup>
+
+### How does Benjamin Graham Define These Two Types of Investors?
+
+**Defensive investors** are focused on preserving their investment capital first and foremost. Their goal is the limit their downside, not get astronomical returns or be the next Warren Buffett. These are typically indexers or people who buy into mutual funds like Vanguard.
+
+**Enterprising investors** are willing to devote more time to managing their investments and executing on investment ideas. In short, they spend more time analyzing all the aspects of a business and get to know it thoroughly. These are the investors who are trying to make higher returns over longer periods of time. Perhaps not becoming billionaires, but they focus on becoming significantly wealthier than your average joe.
+
+### When is it a Good Point to Become an Investor?
+
+Everyone can be a mix, but taking big risks when you're in debt or living hand to mouth is flirting with disaster. The money used for  investing is money set aside for this purpose after all your other needs have been met and you're safe.
+
+Benjamin Graham doesn't go into a ton of detail of this, but I think it is safe to say that these are reasonable criteria starts to a list, meaning more reasons exist but these were the ones I could think of on a Sunday afternoon:
```

Comment:
"reasonable criteria starts to a list" I am not reading this well, is there a mistake? I don't understand.